ext/afform/core/sql/auto_install.sql

```diff
@@ -0,0 +1,45 @@
+-- +--------------------------------------------------------------------+
+-- | Copyright CiviCRM LLC. All rights reserved.                        |
+-- |                                                                    |
+-- | This work is published under the GNU AGPLv3 license with some      |
+-- | permitted exceptions and without any warranty. For full license    |
+-- | and copyright information, see https://civicrm.org/licensing       |
+-- +--------------------------------------------------------------------+
+--
+-- Generated from schema.tpl
+-- DO NOT EDIT.  Generated by CRM_Core_CodeGen
+--
+-- /*******************************************************
+-- *
+-- * Clean up the existing tables - this section generated from drop.tpl
+-- *
+-- *******************************************************/
+
+SET FOREIGN_KEY_CHECKS=0;
+
+DROP TABLE IF EXISTS `civicrm_afform_submission`;
+
+SET FOREIGN_KEY_CHECKS=1;
+-- /*******************************************************
+-- *
+-- * Create new tables
+-- *
+-- *******************************************************/
+
+-- /*******************************************************
+-- *
+-- * civicrm_afform_submission
+-- *
+-- * Recorded form submissions
+-- *
+-- *******************************************************/
+CREATE TABLE `civicrm_afform_submission` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique Submission ID',
+  `contact_id` int unsigned,
+  `afform_name` varchar(255) COMMENT 'Name of submitted afform',
+  `data` text COMMENT 'IDs of saved entities',
+  `submission_date` timestamp DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  CONSTRAINT FK_civicrm_afform_submission_contact_id FOREIGN KEY (`contact_id`) REFERENCES `civicrm_contact`(`id`) ON DELETE SET NULL
+)
+ENGINE=InnoDB ROW_FORMAT=DYNAMIC;
```

Comment:
@seamuslee001 I manually added `ROW_FORMAT=DYNAMIC` but in case you want to tackle getting `civix` to do it...